doc/connection-to-the-internet-of-things/how-to-read-measures-captured-from-iot-devices.md

```diff
@@ -1,121 +1,241 @@
+Context Broker. Ultralight2.0 is selected in this example because of its
+simplicity. In addition we show you how you can do the same locally with the
+*FIWARE Tour Guide Application* as it integrates the [IDAS UL2.0 IoT Agent](https://github.com/telefonicaid/iotagent-ul). 
 
 If you want to quickly connect or simulate virtual devices you may also
-check FIGWAY, a set of simple python scripts working as a client SDK for
-any desktop PC, laptop or gateway supporting a python2.7 environment.
+check [https://github.com/telefonicaid/fiware-figway](FIGWAY),
+a set of simple python scripts working as a client SDK for
+any desktop PC, laptop or gateway supporting a Python 2.7 environment.
 This way you may skip the steps described below and use the python
 commands as described in the README.md file available at the Github
 repository linked above.
 
-You can find a good summary of the APIs described below also at:  
+For a more detailed description of the IDAS (FIWARE IoT Agent suite) APIs,
+please have a look [here](http://docs.telefonicaiotiotagents.apiary.io/).
 
-[http://docs.telefonicaiotiotagents.apiary.io/\#reference/undefined/device/remove-a-device ](http://docs.telefonicaiotiotagents.apiary.io/#reference/undefined/device/remove-a-device)
+A typical IoT data workflow using FIWARE consists of the following steps:
 
-Basically, there are four simple steps to follow:
+Step 1 : **Create an IDAS Service**
 
-**1. Create an IDAS Service**  
- If you are using the public IDAS instance with the public “OpenIoT”
-testing service available at 130.206.80.47 (Port 5073) you may skip this
+  If you are using the public IDAS instance with the public `openiot`
+testing service available at `130.206.80.47` (Port `5073`) you may skip this
 step. Just keep in mind the shared secret for this public service (that
-your devices need to know) is the string “4jggokgpepnvsb2uv4s40d59ov”.  
- Otherwise, creating an IDAS Service is basically a simple HTTP POST
-like this:
+your devices need to know) is the string `4jggokgpepnvsb2uv4s40d59ov`.  
+ Otherwise, creating an IDAS Service consists of a simple HTTP POST like this:
 
-    POST <idas_host>:<idas_port>/iot/services
-    //Example: HTTP POST: http://130.206.80.40:5371/iot/services 
-    Headers: {'content-type': 'application/json’; 'X-Auth-Token' : [TOKEN]; "Fiware-Service: OpenIoT”; "Fiware-ServicePath: /"}
+    POST http://130.206.80.40:5371/iot/services
+    
+    Headers:
+    
+    {
+      'Content-Type':       'application/json',
+      'X-Auth-Token' :      '[TOKEN]',
+      'Fiware-Service':     'openiot',
+      'Fiware-ServicePath': '/'
+    }
+    
     Payload:
+    
     {
       "services": [
         {
-        "apikey": "4jggokgpepnvsb2uv4s40d59ov",
-        "token": "token2",
-        "cbroker": "http://0.0.0.0:1026",
-        "entity_type": "thing",
-        "resource": "/iot/d"
+          "apikey":      "4jggokgpepnvsb2uv4s40d59ov",
+          "token":       "aToken",
```

Comment:
This token refers to Keystone trust tokens, and is an option that it's not commonly used. I think we should remove it from a simple example like this one.